notebook-src/02-book-balancing-intro.rst

```diff
@@ -0,0 +1,298 @@
+=========================================================================================
+2. What Are Mechanical Vibrations?: Investigating a Book Oscillating on a Cylindrical Cup
+=========================================================================================
+
+Introduction
+============
+
+This notebook introduces a single degree of freedom vibratory system in which a
+textbook balances on a cylindrical cup. The system is implemented as a model
+that students can interact with in order to visualize its free response and
+compare the computer simulation to the demonstration in the classroom.
+
+After the completion of this assignment students will be able to:
+
+- view and set parameters on a system
+- visualize a system's free response
+- estimate the period of a sinusoidal vibration from a time series
+- compare a computer simulation result to experimental result
+- interactively adjust the book inertia to see the affect on system response
+- understand the concept of natural frequency nd its relationship to
+  mass/inertia
```

Comment:
Remove the natural frequency, there is no way to extract it from the data alone, only the damping natural frequency.

Reply:
There is no damping in this problem, so maybe we can talk about it.